website/content/en/docs/building-operators/ansible/migration.md

```diff
@@ -1,14 +1,13 @@
 ---
-link: Migrating pre-v1.0.0 Projects
-linkTitle: Migrating pre-v1.0.0 Projects
-weight: 6
-description: Instructions for migrating a Ansible-based project built prior to v1.0.0 to use the new Kubebuilder-style layout.
+link: Migrating Projects from pre-v1.0.0 to the latest release
+linkTitle: Migrating from pre-v1.0.0 to latest
+weight: 200
+description: Instructions for migrating an Ansible-based project built prior to `v1.0.0` (`0.19.x+`) to use the Kubebuilder-style layout which is the default layout adopted by SDK since the `1.0.0` release.
 ---
 
 ## Overview
 
-The motivations for the new layout are related to bringing more flexibility to users and
-part of the process to [Integrating Kubebuilder and Operator SDK][integration-doc].
+The motivation for the new layout is to [integrate Kubebuilder and Operator SDK][integration-doc] based projects. This enables Kubebuilder based projects to use the features provided by Operator SDK. For further information behind this motivation, check [the differences between Kubebuilder and Operator-SDK][what-are-the-the-differences-between-kubebuilder-and-operator-sdk].
```

Comment:
If I'm migrating an operator-sdk project, why do I care about the integration with kubebuilder?
```suggestion
The v1.0 release improves upon prior `operator-sdk` releases with a new project structure and CLI, each of which enhances project extensibility and customizability. These design changes are influenced by [`kubebuilder`](https://book.kubebuilder.io/).
```

Reply:
The point here is to describe why it changes. I think you are right it can be better. 
However, we might need to share the link in the FAQ. Let's see if you like the proposed solution.